src/01-functions.js

```diff
@@ -93,7 +101,9 @@ function average() {}
  *  averageThree(10, 11, 19);
  *  //> 10 // 10 + 11 + 19 = 30; 30 / 3 = 10;
  */
-function averageThree() {}
+ function averageThree(a, b, c) {
+  return (a + b + c)/3
```

Comment:
Let's space our our arithmetic ` (a + b + c) / 3 `.  Very nitcpicky but we want these habits formed as early as possible.